### ISSUE TITLE
initrd is not reproducible

### ISSUE THREAD

**Issue by osresearch** [osresearch/heads#2]

**Comment by osresearch** [comment]
TODO: write a cpio processor that resets all of the dates.

**Comment by osresearch** [comment]
TODO: write a cpio process that also resets the inode numbers since they are being regenerated.
```
 000003c0: 30 30 30 30 30 30 30 30 30 44 30 30 30 30 30 30  000000000D000000
 000003d0: 30 30 62 69 6e 2f 62 61 73 65 6e 61 6d 65 00 00  00bin/basename..
 000003e0: 62 75 73 79 62 6f 78 00 30 37 30 37 30 31 30 30  busybox.07070100
-000003f0: 36 32 30 44 41 43 30 30 30 30 38 31 45 44 30 30  620DAC000081ED00
+000003f0: 36 32 31 30 33 30 30 30 30 30 38 31 45 44 30 30  621030000081ED00
 00000400: 30 30 30 33 45 38 30 30 30 30 30 33 45 38 30 30  0003E8000003E800
 00000410: 30 30 30 30 30 31 30 30 30 30 34 36 35 30 30 30  0000010000465000
 00000420: 30 34 42 46 33 30 30 30 30 30 30 30 30 30 30 30  04BF300000000000
```

**Comment by osresearch** [comment]
Fixed in commit 0646b0bd410804921f3ac560709b7892254b7775
